README.rst

```diff
@@ -365,11 +371,101 @@ You will also have to manually remove a few settings::
+
+mfe-lms-production-settings
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Python-formatted LMS settings in production. Values defined here override the values from `mfe-lms-common-settings <#mfe-lms-common-settings>`_. For an example on the usage of this patch, check out `this section <#mfe-lms-settings>`_.
+
+File changed: ``apps/openedx/settings/lms/production.py``
+
+mfe-lms-common-settings
+~~~~~~~~~~~~~~~~~~~~~~~
+Python-formatted LMS settings used both in production and development.
+
+File changed: ``apps/openedx/settings/partials/common_lms.py``
+
+mfe-webpack-dev-config
+~~~~~~~~~~~~~~~~~~~~~~
+Add any additional configurations to be added at the end of the development webpack config file in Javascript format.
+
+File changed: ``tutormfe/templates/mfe/apps/mfe/webpack.dev-tutor.config.js``
+
+mfe-dockerfile-pre-npm-install
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Add any additional instructions for before the npm install is initiated.
+
+File changed: ``tutormfe/templates/mfe/build/mfe/Dockerfile``
+
+mfe-dockerfile-pre-npm-install-{}
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Add any additional instructions for before the npm install is initiated for a specific mfe. Add the exact mfe name at the end to only change instructions for that mfe.
+
+Example: ``mfe-dockerfile-pre-npm-install-learning`` Will only apply any instructions specified for the learning mfe.
+
+File changed: ``tutormfe/templates/mfe/build/mfe/Dockerfile``
+
+mfe-dockerfile-post-npm-install
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Add any additional instructions for after the npm install has completed. This will apply the instructions to every MFE. For an example on the usage of this patch, check out `here <#mfe-docker-post-npm-install>`_.
+
+File changed: ``tutormfe/templates/mfe/build/mfe/Dockerfile``
+
+mfe-dockerfile-post-npm-install-{}
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Add any additional instructions for after the npm install has completed for a specific mfe. Add the exact mfe name at the end to only change instructions for that mfe. For an example on the usage of this patch, check out `here <#mfe-docker-post-npm-install>`_.
+
+Example: ``mfe-dockerfile-post-npm-install-authn`` Will only apply any instructions specified for the authn mfe.
+
+File changed: ``tutormfe/templates/mfe/build/mfe/Dockerfile``
+
+mfe-dockerfile-pre-npm-build
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Add any additional instructions for before the build step initializes. This will apply the instructions to every MFE. For an example on the usage of this patch, see `over here <#mfe-docker-pre-npm-build>`_.
+
+File changed: ``tutormfe/templates/mfe/build/mfe/Dockerfile``
+
+mfe-dockerfile-pre-npm-build-{}
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Add any additional instructions for before the build step initializes for a specific mfe. Add the exact mfe name at the end to only change instructions for that mfe. For an example on the usage of this patch, see `over here <#mfe-docker-pre-npm-build>`_.
```

Comment:
a specific mfe -> a specific MFE
for that mfe -> for that MFE

(same for other occurrences of "mfe" in the document)